QgisModelBaker/libqgsprojectgen/dataobjects/project.py

```diff
@@ -115,13 +118,19 @@ def create(self, path: str, qgis_project: QgsProject):
             referencing_field_constraints = referencing_layer.fieldConstraints(rel.referencingFields()[0])
             allow_null = not bool(referencing_field_constraints & QgsFieldConstraints.ConstraintNotNull)
 
+            # If we have an extended ili2db domain, we need to filter its values
+            filter_expression = "\"{}\" = '{}'".format(ENUM_THIS_CLASS_COLUMN,
+                                                       relation.child_domain_name) if relation.child_domain_name else ''
```

Comment:
if there is no `relation.child_domain_name` we still enter something into the filterExpression? Or am I missing something?

Reply:
Ah sorry. I misread the `if`. Everything is fine...